main/runner.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
+
+# Copyright 2015 Google Inc
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+# [START runner]
+"""App Engine local test runner example.
+
+This program handles properly importing the App Engine SDK so that test modules
+can use google.appengine.* APIs and the Google App Engine testbed.
+
+Example invocation:
+
+    $ python runner.py ~/google-cloud-sdk
+"""
+
+import argparse
+import os
+import sys
+import unittest
+
+
+def fixup_paths(path):
+    """Adds GAE SDK path to system path and appends it to the google path
+    if that already exists."""
+    # Not all Google packages are inside namespace packages, which means
+    # there might be another non-namespace package named `google` already on
+    # the path and simply appending the App Engine SDK to the path will not
+    # work since the other package will get discovered and used first.
+    # This emulates namespace packages by first searching if a `google` package
+    # exists by importing it, and if so appending to its module search path.
+    try:
+        import google
+        google.__path__.append("{0}/google".format(path))
```

Comment:
use two space for indentation and single quote for strings

Reply:
Right.. I would still match the styling of it.. since it's inside the main directory..

Reply:
Done. I've left the docstrings with double quotes as these seem to be standard and I couldn't see any examples elsewhere in the codebase to compare...